lib/tasks/evm_settings.rake

```diff
@@ -81,16 +81,19 @@ module EvmSettings
   private_class_method :keyval_to_hash
 
   def self.str_to_value(value)
-    return true  if value =~ /true/i
-    return false if value =~ /false/i
-    return nil   if value =~ /nil/
+    return value unless value.kind_of?(String)
+    return true  if value.match(/true/i)
+    return false if value.match(/false/i)
+    return nil   if value.match(/nil/)
```

Comment:
```suggestion
    return true  if value.match?(/true/i)
    return false if value.match?(/false/i)
    return nil   if value.include?("nil")
```

Alternatively, for the true/false case, you can do `value.downcase.include?("false")`, which is cheaper than a regex.